tests/functional/pv/pv_services/test_cephfilesystem_creation.py

```diff
@@ -39,17 +38,14 @@ def test_Cephfilesystem_creation(self):
                 logger.info("CephFile System Created. : test-ceph-fs")
             else:
                 logger.error("Unable to create the Ceph File System")
-            ct_pod = pod.get_ceph_tools_pod()
-            cmd1 = "ceph fs fail test-ceph-fs"
-            ct_pod.exec_cmd_on_pod(cmd1)
-            cmd2 = "ceph fs rm test-ceph-fs --yes-i-really-mean-it"
-            ct_pod.exec_cmd_on_pod(cmd2)
+            logger.info("Deleting the Cephf Filesystem")
+            cephFS_obj.delete()
```

Comment:
After cephFS_obj.delete(), I think we should add a check to wait for Pod removal before adding it again. Probably that's the leftover Pod being reported.

Reply:
@shivamdurgbuns Please verify that the cephfilesystem is actually deleted.

```
cephFS_obj.ocp.wait_for_delete(
                cephFS_obj.name
            )
```

Reply:
okay will check it out